GenderPayGap.WebUI/Search/SearchRepository.cs

```diff
@@ -82,6 +86,8 @@ private List<string> ExtractSearchTermsFromQuery(string query)
                 .ToList();
         }
 
+
+        #region Search Organisations
         private List<AdminSearchResultOrganisationViewModel> SearchOrganisations(string query, List<string> searchTerms)

```

Comment:
The `#regions` do a nice job of providing a narrative, but I think the same is achieved just with the method names here. Which makes the regions feel a bit redundant